torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2233,21 +2260,37 @@ RegisterOperators reg2({
     CREATE_LIST_OPS("t", c10::List<IValue>),
 #undef CREATE_LIST_OPS
     Operator(
-        "aten::sort(int[](a!) self) -> ()",
+        "aten::sort(int[](a!) self, bool reverse=False) -> ()",
         listSort<int64_t>,
         aliasAnalysisFromSchema()),
     Operator(
-        "aten::sort(float[](a!) self) -> ()",
+        "aten::sort(float[](a!) self, bool reverse=False) -> ()",
         listSort<double>,
         aliasAnalysisFromSchema()),
     Operator(
-        "aten::sort(Tensor[](a!) self) -> ()",
+        "aten::sort(Tensor[](a!) self, bool reverse=False) -> ()",
         listSort<at::Tensor>,
         aliasAnalysisFromSchema()),
     Operator(
-        "aten::sort(bool[](a!) self) -> ()",
+        "aten::sort(bool[](a!) self, bool reverse=False) -> ()",
         listSort<bool>,
         aliasAnalysisFromSchema()),
+    Operator(
+        "aten::sorted(int[](a) input) -> (int[])",
```

Comment:
The difference between `sort` and `sorted` isn't very obvious, how about `aten::sort` and `aten::sort_` instead?

Reply:
This is a mapping to a python builtin. It would also be confusing because you would assume `aten::sort_` mutates the input, whereas `sorted` copies the input and sorts it